example/package.json

```diff
@@ -2,21 +2,22 @@
   "name": "example",
   "version": "0.0.0",
   "scripts": {
-    "dev": "vite",
+    "dev": "vite --host",
```

Comment:
```suggestion
    "dev": "vite",
```

invalid usage

```
--host [host]           [string] specify hostname
```

Reply:
This might be something different with Vite 3, but if you look here it says you can do as you are suggesting. But what I want is to see the usual `localhost:XXXX` rather than `1XX.xX.XX:XXXX` pattern. 
See [here](https://vitejs.dev/config/server-options.html#server-host)

Reply:
OK. Whether or not to use that feature is environment-dependent, so it's best not to include it in the npm script; if you want to use host, just use `yarn dev -host`.

Reply:
Ok. I'll remove it just to be safe and then merge